src/integration-test/integration/ReportsClient.test.ts

```diff
@@ -105,14 +105,6 @@ describe("Reports Client", () => {
     expect(report.description).to.be.eq("Updated");
   });
 
-  it("Reports - Get all non deleted", async () => {
```

Comment:
Why did you remove this test?

Reply:
Not sure how to correctly test this because if other tests create reports this test becomes polluted with them. Perhaps I should just check if the count is greater than 3?

Reply:
Added test back.